### PR TITLE
Access Control Podcast: #14. Rob Zuber

### PR DESCRIPTION
**Pull request by benarent**
Preview: https://teleport-docs-next-lsv6k9f5k-gravitational.vercel.app/resources/podcast/access-control-robert-zuber/

### CODE REVIEW

File: pages/resources/podcast/access-control-robert-zuber/index.mdx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+previewImage: "og-cards/podcast/14_rob_zuber.png"
+podcastName: 14-secure-ci-cd
+layout: podcast
+publicationDate: "2022-04-18"
+---
+
+
+# Securing CI/CD and the Software Supply Chain.
+
+For this 14th episode of Access Control Podcast, a podcast providing practical security advice for startups, Developer Relations Manager at Teleport Ben Arent chats with Rob Zuber. Rob is a seasoned entrepreneur. And along his startup journey, he ran into issues with mobile CI/CD. This led him to create Distiller, a CI/CD service focused for mobile devices. The Distiller team later joined forces with CircleCI, and Rob has been CircleCI's CTO for the last 7 years. [CircleCI](https://circleci.com/) is a service that provides build and deployment infrastructure for modern apps. CirceCI is a hyper-growth startup that also supports many startups.
```
Comment (dumez-k) [inline pages/resources/podcast/access-control-robert-zuber/index.mdx:13]:
```suggestion
For this 14th episode of Access Control Podcast, a podcast providing practical security advice for startups, Developer Relations Manager at Teleport Ben Arent chats with Rob Zuber. Rob is a seasoned entrepreneur and along his startup journey, he ran into issues with mobile CI/CD. This led him to create Distiller, a CI/CD service focused on mobile devices. The Distiller team later joined forces with CircleCI, and Rob has been CircleCI's CTO for the last 7 years. [CircleCI](https://circleci.com/) is a service that provides build and deployment infrastructure for modern apps. CirceCI is a hyper-growth startup that also supports many startups. 
```

File: pages/resources/podcast/access-control-robert-zuber/index.mdx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+ - With regard to software supply chain problems, as with other similar problems, there's always the question of how long have we known about something versus how long has it been happening. 
+ - Continuous deployment is important for remediation because the length of time to push a deployment impacts the duration of exposure to a given security problem.
+ - The SolarWinds incident was caused by a compromised build server and involved sophisticated loading of a backdoor into the deployed Orion system. 
+ - Prior to recent security incidents, tradional CI/CD's focussed around image and artifact scanning. Securing Tokens and Build Infrastructure have been a key part of the solution to keep CI/CD secure.  
+ - As companies string together a large number of tools, it's important for them to ask: What is the security model we have here?  We'll dicuss this in detail with this eposide. 
+
+
+### Expanding your knowledge on Access Control Podcast: Episode 14 - Securing CI/CD and Supply Chain
+ 
+ - [CirclCI.com](https://circleci.com/)
```
Comment (dumez-k) [inline pages/resources/podcast/access-control-robert-zuber/index.mdx:27]:
```suggestion
 - [CircleCI.com](https://circleci.com/)
```

File: pages/resources/podcast/access-control-robert-zuber/index.mdx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+ - [Teleport 9 - Introducing Machine ID](https://goteleport.com/blog/machine-to-machine-access/)
+ - [Highly Evasive Attacker Leverages SolarWinds Supply Chain to Compromise Multiple Global Victims With SUNBURST Backdoor](https://www.mandiant.com/resources/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor)
+ - [Project Trebuchet: How SolarWinds is Using Open Source to Secure Their Supply Chain](https://www.youtube.com/watch?v=1-tMRxqMwTQ)
+ - [Teleport Machine ID](use-cases/machine-to-machine-access/)
```
Comment (dumez-k) [inline pages/resources/podcast/access-control-robert-zuber/index.mdx:31]: This link is broken on preview

**Review by dumez-k — CHANGES_REQUESTED** [review]

File: pages/resources/podcast/access-control-robert-zuber/index.mdx (hunk 3 above)
Comment (dumez-k) [reply pages/resources/podcast/access-control-robert-zuber/index.mdx:31]: But should be fixed when the site is live?

Comment (C-STYR) [reply pages/resources/podcast/access-control-robert-zuber/index.mdx:31]:
```suggestion
 - [Teleport Machine ID](https://goteleport.com/use-cases/machine-to-machine-access/)
```
we should just default to absolute urls, there isn't really a good reason to use relative ones

**Review by C-STYR — APPROVED** [review]

**Review by dumez-k — APPROVED** [review]

**Pull request merged**